lib/Settings.js

```diff
@@ -13,6 +14,20 @@ var Settings = {};
 
 var repositories = {};
 
+function dumpEnvironment(obj, parentName) {
+    var envString = '';
+    Object.keys(obj).forEach(function(name) {
+        var fullName = parentName === '' ? name : parentName + '__' + name;
```

Comment:
Is there a way to do this with `nconf` directly? Like https://www.npmjs.com/package/nconf#file?

Reply:
Yeah I was looking at that. I believe that outputs it as a JSON whereas we would need to convert it back to a flat list joined with `__` anyway.